packages/cfd/src/Components/props.types.ts

```diff
@@ -297,3 +297,24 @@ export type TDetailsOfEachMT5Loginid = DetailsOfEachMT5Loginid & {
 
     openFailedVerificationModal?: (from_account: string) => void;
 };
+
+export type TSuccessDialogProps = {
+    is_open: boolean;
+    toggleModal: () => void;
+    onCancel?: () => void;
+    onSubmit?: () => void;
+    classNameMessage?: string;
+    message: string | React.ReactElement;
+    icon: React.ReactElement;
+    icon_size?: string;
+    text_submit?: string;
+    text_cancel?: string;
+    heading?: string | React.ReactElement;
+    title?: string;
+    icon_type?: string;
+    is_medium_button?: boolean;
+    has_close_icon: boolean;
+    width?: string;
+    has_cancel?: boolean;
+    has_submit?: boolean;
+};
```

Comment:
can we move this to success-dialog component?

Reply:
At the time of typescript migration, what I had on my mind was to put types in a separate file and just import them in the component that is using it. Also we needed to have a shared types file to import them in other files instead of repeating them to only keep logic in the component and keep types in a separate file. I saw this in other projects as well. Since the migration was done by different developers at that time, we do not have that consistency in our code base which can be improved.
What do you think about this ?
@matin-deriv @farrah-deriv

Reply:
But these types are only being used in 1 component. It is not shared across the components. Why do we need to have it in a separate file?

Reply:
1 components type file, 1 containers type file and one for shared. This one is added to the components types file.

Reply:
But what's the purpose of having a separate file for components/containers types?

Reply:
moved the types to component